core/dbt/contracts/graph/node_args.py

```diff
@@ -21,6 +21,7 @@ class ModelNodeArgs:
     generated_at: datetime = field(default_factory=datetime.utcnow)
     depends_on_nodes: List[str] = field(default_factory=list)
     enabled: bool = True
+    time_spine: Optional[TimeSpine] = None
```

Comment:
Don't think this one ([and its usage below](https://github.com/dbt-labs/dbt-core/pull/10483/files#diff-42cfc99b2fcd89ab75957b8999bee2b1af822f7d39953ca99f1f285c170557c1R465)) is necessary as these represent external node fields

Reply:
Removed!